src/onelogin/saml2/response.py

```diff
@@ -128,14 +128,15 @@ def is_valid(self, request_data, request_id=None):
                     raise Exception('There is an EncryptedAttribute in the Response and this SP not support them')
 
                 # Checks destination
-                destination = self.document.get('Destination', '')
-                if destination:
-                    if not destination.startswith(current_url):
-                        # TODO: Review if following lines are required, since we can control the
-                        # request_data
-                        #  current_url_routed = OneLogin_Saml2_Utils.get_self_routed_url_no_query(request_data)
-                        #  if not destination.startswith(current_url_routed):
-                        raise Exception('The response was received at %s instead of %s' % (current_url, destination))
+                if security.get('checkDestination', True):
```

Comment:
Check the destination is a MUST on the SAML standard so I can't make it optional.

SAML Bindings spec contains the following under security considerations for Redirect and POST bindings:
```
If the message is signed, the Destination XML attribute in the root SAML element of the protocol message MUST contain the URL to which the sender has instructed the user agent to deliver the message. The recipient MUST then verify that the value matches the location at which the message has been received.
```